assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.stories.js

```diff
@@ -73,6 +77,10 @@ PendingVerification.storyName = 'PendingVerification';
 PendingVerification.scenario = {};
 PendingVerification.args = {
 	setupRegistry: ( registry ) => {
+		publications[ 1 ].products.push( {
```

Comment:
Similar to my comment above, we shouldn't mutate objects which are in the Redux store.

As the publication is passed to `selectPublication()`, which destructures its properties (so it's not doing a referential check), we can solve this by cloning the publication before mutating its properties. The `cloneDeep()` lodash utility can be used to clone the object.

Please apply this to the other updated stories where relevant.